pom.xml

```diff
@@ -8,34 +8,77 @@
     <groupId>com.microsoft.azure</groupId>
     <artifactId>kafka-sink-azure-kusto</artifactId>
     <packaging>jar</packaging>
+    <description>A Kafka Connect plugin for Azure Kusto Database</description>
     <version>1.0.1</version>
+    <properties>
+        <kafka.version>1.0.0</kafka.version>
+        <json.version>20090211</json.version>
+        <commonio.version>2.6</commonio.version>
+        <kusto-sdk.version>2.1.0</kusto-sdk.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <failsafe.version>2.21.0</failsafe.version>
+        <junit.surefire.plugin.version>1.2.0</junit.surefire.plugin.version>
+    </properties>
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-assembly-plugin</artifactId>
-                <configuration>
```

Comment:
can i ask why did u dropped this ? it just provided a clean way to create one jar instead of the list of jars ..